packages/components/pie-button/src/index.ts

```diff
@@ -1,17 +1,48 @@
 import { LitElement, html, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
 import { customElement, property } from 'lit/decorators.js';
-import styles from './styles.scss?inline';
+import styles from './button.scss?inline';
+import { BUTTON_VARIANT, BUTTON_TYPE } from './defs';
+import { validValues } from './decorators';
+
+// Valid values available to consumers
+export { BUTTON_VARIANT, BUTTON_TYPE };
+
+const validButtonVariants = Object.values(BUTTON_VARIANT);
+const validButtonTypes = Object.values(BUTTON_TYPE);
```

Comment:
Is there some limitation or use case that prevents us from passing the enum, so we don't need to evaluate `Object.values(...)` for each and every component prop? Those consts don't have any other role than validation.

Reply:
I was discussing with @jamieomaguire yesterday about the possiblity of exporting arrays as well as the enums, so we wouldn't have to do this.

Reply:
We could probably move that into the decorator and just pass in the enum?